x/feemarket/keeper/params.go

```diff
@@ -16,3 +17,24 @@ func (k Keeper) GetParams(ctx sdk.Context) (params types.Params) {
 func (k Keeper) SetParams(ctx sdk.Context, params types.Params) {
 	k.paramSpace.SetParamSet(ctx, &params)
 }
+
+// ----------------------------------------------------------------------------
+// Parent Base Fee
+// Required by EIP1559 base fee calculation.
+// ----------------------------------------------------------------------------
+
+// GetConstantFee get's the base fee from the paramSpace
+// return nil if base fee is not enabled
+func (k Keeper) GetBaseFee(ctx sdk.Context) *big.Int {
+	params := k.GetParams(ctx)
+	if params.NoBaseFee {
+		return nil
+	}
+
+	return params.BaseFee.BigInt()
+}
+
+// SetBaseFee set's the base fee in the paramSpace
+func (k Keeper) SetBaseFee(ctx sdk.Context, baseFee *big.Int) {
+	k.paramSpace.Set(ctx, types.ParamStoreKeyBaseFee, sdk.NewIntFromBigInt(baseFee))
+}
```

Comment:
remove and use `SetParams` instead

Reply:
This is a shortcut for 

```
params := k.getParams(ctx)
params.BaseFee = sdk.NewIntFromBigInt(baseFee)
SetParams(params)
```

Is there any issue using it?

Reply:
no, I guess it's fine